ili9341/spi_atsamd51.go

```diff
@@ -0,0 +1,72 @@
+// +build atsamd51
+
+package ili9341
+
+import (
+	"machine"
+)
+
+type spiDriver struct {
+	bus machine.SPI
+	dc  machine.Pin
+	rst machine.Pin
+	cs  machine.Pin
+	rd  machine.Pin
+}
+
+func NewSpi(bus machine.SPI, dc, cs, rst machine.Pin) *Device {
+	return &Device{
+		dc:  dc,
+		cs:  cs,
+		rst: rst,
+		rd:  machine.NoPin,
+		driver: &spiDriver{
+			bus: bus,
+		},
+	}
+}
+
+func (pd *spiDriver) configure(config *Config) {
+}
+
+//go:inline
```

Comment:
Do all these `//go:inline` annotations really improve performance?

In general they should only be added as a fix for performance. Also, before doing that please test with `-opt=2` first because that will likely make the `//go:inline` pragmas unnecessary (and you should add `-opt=2` if speed is important).

Reply:
go:inline is something I copied from parallel_atsamd51.go.
I wasn't particularly aware of it.

Removing go:inline didn't change the speed.
So, I'll remove it.

Reply:
The FPS didn't change when I set -opt=2.